package.go

```diff
@@ -11,6 +12,7 @@ import (
 
 // A Package holds the typedef, function, and enum definitions for a Go package.
 type Package struct {
+	Prefix  string
 	Name    string
```

Comment:
I've checked, and there was not a single unused struct fields. They are all still used (mostly by `UniqueName()`).

I did fix a typo in a comment, but that's all.